go/vt/vttablet/tabletserver/throttle/throttler_test.go

```diff
@@ -555,7 +563,11 @@ func TestReplica(t *testing.T) {
 	runThrottler(t, ctx, throttler, time.Minute, func(t *testing.T, ctx context.Context) {
 		assert.Empty(t, tmClient.AppNames())
 		flags := &CheckFlags{}
-		throttler.CheckByType(ctx, throttlerapp.VitessName.String(), "", flags, ThrottleCheckSelf)
+		{
+			checkResult := throttler.CheckByType(ctx, throttlerapp.VitessName.String(), "", flags, ThrottleCheckSelf)
+			assert.False(t, checkResult.RecentlyChecked) // "vitess" app does not mark the throttler as recently checked
+			assert.False(t, throttler.recentlyChecked()) // "vitess" app does not mark the throttler as recently checked
```

Comment:
This one isn't by app name, is it?

Reply:
Not this one. This is a general purpose "has anyone been bothering this throttler"